packages/unmock-core/src/__tests__/service.util.test.ts

```diff
@@ -109,41 +113,63 @@ describe("Tests getPathParametersFromSchema", () => {
 
 describe("Tests buildPathRegexStringFromParameters", () => {
   const emptySchema: any[] = [];
-  const schemaWithMatchingParams = [
-    {
-      0: { in: "path", name: "petId" },
-      1: { in: "query", name: "foo" },
-    },
+  const schemaWithMatchingParams: Parameter[] = [
+    { in: "path", name: "petId" },
+    { in: "query", name: "foo" },
   ];
-  const schemaWithMissingParams = [
-    {
-      0: { in: "path", name: "id" },
-      1: { in: "query", name: "foo" },
-    },
+  const schemaWithMissingParams: Parameter[] = [
+    { in: "path", name: "id" },
+    { in: "query", name: "foo" },
   ];
 
   it("Tests path", () => {
     // Empty schema - nothing to do
-    expect(buildPathRegexStringFromParameters("/pets", emptySchema, [])).toBe("/pets");
+    expect(buildPathRegexStringFromParameters("/pets", emptySchema, [])).toBe(
+      "/pets",
+    );
```

Comment:
Here it would be possible to follow the "one assert per test rule" by putting each of these expects in a unit test of their own, under a common `describe()`. But that's just a style thing.